### PR TITLE
 Revampt NFT-related UIs

### PR DESCRIPTION
**Pull request by lth08091998**

### CODE REVIEW

**Review by lth08091998 — COMMENTED** [review]
image

- Should highlight the selected section ("Markets" or "Mint NFT")

**Comment by lth08091998** [comment]
@winbrother  image

Placing the drop-down here is a bit strange

**Comment by lth08091998** [comment]
@winbrother Similar to OpenSea, maybe separate the field name and field box separately like this?

image

File: pages/marketplace/CreateNftAssetSection.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
+import { ethers } from 'ethers'
```
Comment (lth08091998) [inline pages/marketplace/CreateNftAssetSection.tsx:1]: @winbrother  `CreateNftAssesSection`, `MarketList`, `MarketplaceMenu` (those components not representing the whole page), should be placed in `components` directory

Comment (winbrother) [reply pages/marketplace/CreateNftAssetSection.tsx:1]: I plan to move shared components to that folder once everything is finalized. Once the project scale up, placing every single components into 1 folder make it very hard to trace (based on my experience). Shared component should be navbar, footer, image component and the like.

Comment (lth08091998) [reply pages/marketplace/CreateNftAssetSection.tsx:1]: Can put those to `components/asset` e.g. As long as they are not in `pages` folder.

**Pull request merged**